components/UI/ServiceTable.js

```diff
@@ -67,7 +67,7 @@ const TableHeader = styled.th`
   line-height: calc(var(--margin) * 3);
   text-align: ${props => props.align && props.align};
   white-space: nowrap;
-
+  ${props => props.width && `width: ${ typeof props.width === 'string' ? props.width : '8ch'};`}
```

Comment:
Hmm... the `<ServiceTable` is supposed to be a super generic and everywhere-to-be-used component, right?
Correct me if I am wrong, but with this change we'd tell every table header to be `8ch` wide, unless we specify a custom width, no?

And all the new usages of `ServiceTable.Header` as of this PR are providing a custom width. So, why even bother?